Assets/Scripts/Game/UserInterface/SkillsRollout.cs

```diff
@@ -146,7 +146,7 @@ public void Reroll()
             UpdateSkillValueLabels();
         }
 
-        public void SetClassSkills(DFCareer dfClass)
+        public void SetClassSkills(DFCareer dfClass, bool doReroll = true)
```

Comment:
After 1.0, we have to be careful with modifying the signature of public functions. This breaks precompiled mods

https://learn.microsoft.com/en-us/dotnet/standard/library-guidance/breaking-changes#binary-breaking-change
> A binary breaking change happens when you change the public API of your library, so assemblies compiled against older versions of your library are no longer able to call the API. For example, **changing a method's signature by adding a new parameter** will cause assemblies compiled against the older version of the library to throw a [MissingMethodException](https://learn.microsoft.com/en-us/dotnet/api/system.missingmethodexception).

I think in this case, we can avoid it by simply overloading the function instead. Keep the old signature, add a new version with the bool parameter, and the old function simply calls `SetClassSkills(dfClass, true);`

Reply:
Ok, added the old method back in.